playwright-tests/tests/dashboard/home-page.spec.js

```diff
@@ -155,6 +163,35 @@ test.describe("Dashboard Page", function () {
     await page.waitForTimeout(5_000);
     await expect(page.getByText("Please wait a moment...")).toBeVisible();
   });
+
+  test("Shouldn't show treasuries dropdown", async ({
+    page,
+    instanceAccount,
+  }) => {
+    test.setTimeout(60_000);
+    await mockUserDaos({ page, accountId: "megha19.near" });
+    await page.waitForTimeout(3_000);
+    const logo = instanceAccount.includes("devdao")
+      ? page.locator("g rect")
+      : page.locator(".image-container > .rounded-3");
+    await expect(logo).toBeVisible();
+    const treasuriesDropdown = page.locator(".custom-dropdown > .d-flex > .bi");
+    await expect(treasuriesDropdown).toBeHidden();
+  });
+
+  test("Should show user's treasuries dropdown", async ({ page }) => {
```

Comment:
is it not supposed to show the dropdown if it is only one treasury? maybe the title could explain in which scenario you don't want to show the dropdown?

Reply:
yeah updated test description